### PR TITLE
Add custom error pages

### PR DESCRIPTION
**Pull request by dibenzepin**
This should deal with #43.

### CODE REVIEW

**Comment by dibenzepin** [comment]
just realised that hugo/zola export probably needs consideration too

**Comment by dibenzepin** [comment]
not sure why it took me forever but i finally got around to it

**Comment by sirodoht** [comment]
That's awesome, thanks for submitting this! I will simplify the texts slightly but other than that great! 👍

File: docs/src/coding-conventions.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Coding Conventions
 
 1. All files should end with a new line character.
-1. Python code should be formatted with [black](https://github.com/psf/black).
+1. Python code should be formatted with [ruff](https://github.com/astral-sh/ruff).
```
Comment (sirodoht) [inline docs/src/coding-conventions.md:4]: 👌

**Comment by dibenzepin** [comment]
closes #43

**Pull request merged**